tests/beaker_tests/cisco_vxlan_vtep/test_vxlan_vtep.rb

```diff
@@ -27,6 +27,7 @@
 tests = {
```

Comment:
As this is an existing file

` # Copyright (c) 2015 - 2018 Cisco and/or its affiliates.`